environment.yml

```diff
@@ -98,7 +98,7 @@ dependencies:
 
   - pyqt>=5.9.2  # pandas.read_clipboard
   - pytables>=3.4.2  # pandas.read_hdf, DataFrame.to_hdf
-  - s3fs  # pandas.read_csv... when using 's3://...' path
+  - s3fs  # pandas.read_csv... when using 's3://...' path (also brings in fsspec)
```

Comment:
I think explicitly list fsspec here, if we're import it.

What should we set the minimum supported version to? We'll add this to a few places in the library (`compat._optional`, docs, ...).

Reply:
Can update this, and the minimum versions?